administrator/components/com_userlogs/models/fields/extension.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_userlogs
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE
+ */
+
+defined('_JEXEC') or die;
+JFormHelper::loadFieldClass('list');
+/**
+ * Field to load a list of all users that have logged actions
+ */
+class JFormFieldExtension extends JFormFieldList
+{
+
+    /**
+    * The form field type.
+    *
+    * @var     string
+    * @since   1.6
+    */
+   protected $type = 'extension';
+
+   public function getOptions()
+   {
+       $db = JFactory::getDbo();
+       $query = $db->getQuery(true);
+       $query->select('DISTINCT b.extension');
```

Comment:
Do you really need to use distinct here? With a lot of log entries this is maybe an issue.